sympy/parsing/tests/test_latex_lark.py

```diff
@@ -593,3 +789,26 @@ def test_miscellaneous_expressions():
     for latex_str, sympy_expr in MISCELLANEOUS_EXPRESSION_PAIRS:
         with evaluate(False):
             assert parse_latex_lark(latex_str) == sympy_expr, latex_str
+
+
+def test_literal_complex_number_expressions():
+    for latex_str, sympy_expr in UNEVALUATED_LITERAL_COMPLEX_NUMBER_EXPRESSION_PAIRS:
+        with evaluate(False):
+            assert parse_latex_lark(latex_str) == sympy_expr, latex_str
+
+
+# BAD PAIRS:
+#
+# (r"\begin{pmatrix}x & y \\a & b\end{pmatrix}/"
+#  r"\begin{pmatrix}a & b \\x & y\end{pmatrix}"),
+# r"\end{pmatrix}x & y \\a & b\end{pmatrix}",
+# r"\begin{pmatrix}x & y \\a & b\end{pmatrix}\begin{pmatrix}",
+# r"\begin{pmatrix}x & y \\a & b\end{pmatrix}\begin{pmatrix}x & y \\a & b",
+# r"\begin{pmatrix}x & y \\a & b\end{bmatrix}"
```

Comment:
I'm assuming that this was a copy-paste error.

Reply:
I will remove it now.

Reply:
It looks like it has been removed, so I'll close this comment.